### PR TITLE
Fix config catalog issues | Update JWT issuer config

### PR DESCRIPTION
**Pull request by dushaniw**
Fix https://github.com/wso2/docs-apim/issues/1821 and  https://github.com/wso2/docs-apim/issues/1893 related to config catalog.

Fix https://github.com/wso2/docs-apim/issues/2172 to add steps on adding multiple JWT issuers.

### CODE REVIEW

File: en/tools/config-catalog-generator/data/configs.json [hunk 1]
```diff
@@ -1639,6 +1708,67 @@
...
+                            "possible": "",
+                            "description": "Indexing frequency in seconds."
+                        },
+                        {
+                            "name": "bach_size",
+                            "type": "string",
+                            "required": false,
+                            "default": "40",
+                            "possible": "",
+                            "description": "Number of resources submit for given indexing thread."
```
Comment (ruthryi) [inline en/tools/config-catalog-generator/data/configs.json:1750]: "submitted"

Comment (dushaniw) [reply en/tools/config-catalog-generator/data/configs.json]: fixed https://github.com/wso2/docs-apim/pull/2627/commits/1f289d655cebb909b5d1c25190807ee3da98c297

File: en/tools/config-catalog-generator/data/configs.json [hunk 2]
```diff
@@ -1639,6 +1708,67 @@
...
+                            "possible": "",
+                            "description": "Number of worker threads for indexing."
+                        },
+                        {
+                            "name": "re_indexing",
+                            "type": "string",
+                            "required": false,
+                            "default": "Not defined",
+                            "possible": "",
+                            "description": "Provide a number starting from 1. Increase the value by one during each time you need to reindex. After changing the configuration and before restarting the server, make sure to backup and delete the <APIM_HOME>/solr directory. Reindexing may get a considerable amount of time to finish after a server restart, depending on the number of APIs you have in the registry."
```
Comment (ruthryi) [inline en/tools/config-catalog-generator/data/configs.json:1766]: "Make sure to backup and delete the <API-M_HOME>/solr directory after changing the configuration and before restarting the server. After a server restart, reindexing might take a considerable amount of time depending on the number of APIs you have in the registry."

File: en/tools/config-catalog-generator/data/configs.json [hunk 3]
```diff
@@ -1639,6 +1708,67 @@
...
             "exampleFile": "apim.notification.toml"
-        }
+        },
+        {
+            "title": "Registry indexing configurations",
+            "options": [
+                {
+                    "name": "indexing",
+                    "required": false,
+                    "description": "Index configurations related to meta data search in registry.",
```
Comment (ruthryi) [inline en/tools/config-catalog-generator/data/configs.json:1718]: "metadata" is one word

**Review by ruthryi — CHANGES_REQUESTED** [review]
Kindly regenerate the config-catalog.md

File: en/tools/config-catalog-generator/data/configs.json (hunk 3 above)
Comment (dushaniw) [reply en/tools/config-catalog-generator/data/configs.json]: fixed with https://github.com/wso2/docs-apim/pull/2627/commits/3660eecc716fd15449381c42867da33b605f0a1f

File: en/tools/config-catalog-generator/data/configs.json (hunk 2 above)
Comment (dushaniw) [reply en/tools/config-catalog-generator/data/configs.json]: fixed with https://github.com/wso2/docs-apim/pull/2627/commits/3660eecc716fd15449381c42867da33b605f0a1f

**Comment by dushaniw** [comment]
> Kindly regenerate the config-catalog.md

@ruthryi As per @shaniR we will be generating the config-catalog.md at the end of hackathon.

**Review by ruthryi — APPROVED** [review]

**Review by isharac — APPROVED** [review]

**Pull request merged**